regolamento/index.html

```diff
@@ -0,0 +1,162 @@
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/brands.min.css"/>
+    <link rel="shortcut icon" type="image/png" href="/img/favicon.png"/>
+    
+    <!-- This is to convert markdown into html -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/showdown/2.1.0/showdown.min.js" integrity="sha512-LhccdVNGe2QMEfI3x4DVV3ckMRe36TfydKss6mJpdHjNFiV07dFpS2xzeZedptKZrwxfICJpez09iNioiSZ3hA==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+    
+    <script>
+        document.addEventListener('DOMContentLoaded', (event) => {
+            var tg = document.querySelectorAll('.tg-link');
+            for (var i = 0; i < tg.length; i++) {
+                var urlpartsplit = 'hat';
+                var urlpart = 'joinc' + urlpartsplit;
+                var urlpartagain = 't.';
+                var urlpartanother = 'me/';
+                var id = tg[i].getAttribute('data-id');
+                tg[i].setAttribute('href', 'https://' + urlpartagain + urlpartanother + urlpart + '/' + id);
+            }
+        });
+    </script>
+</head>
+<body>
+<div id="top-bar">
+    <div class="container">
+        <div class="row">
+            <ul>
+              <li><a href="/">Home</a></li>
+              <li><a href="/contatti">Contatti</a></li>
+              <li><a href="/download">Download</a></li>
+              <li><a href="https://community.mozilla.org/groups/mozilla-italia/?view=events">Eventi</a></li>
+              <li><a href="/faq">FAQ</a></li>
+              <li><a href="https://forum.mozillaitalia.org/">Forum</a></li>
+            </ul>
+        </div>
+    </div>
+</div>
+<section id="main-logo">
+    <div class="container">
+        <div class="d-flex justify-content-center">
+            <a href="/">
+                <img class="main-logo" src="/img/logo.svg" alt="Logo di Mozilla Italia">
+            </a>
+        </div>
+    </div>
+</section>
+
+<section>
+    <div class="container">
+        <div class="row">
+            <h1 class="title">Regolamento dei gruppi Mozilla Italia</h1>
+            <div id="content"></div>
+
+
+            <script>
+                (function() {
+                    var httpRequest;
+                    makeRequest();
+                
+                    function makeRequest() {
+                        httpRequest = new XMLHttpRequest();
+                
+                        if (!httpRequest) {
+                            alert('Non riesco a creare una XMLHttpRequest');
+                            return false;
+                        }
+                
+                        httpRequest.onreadystatechange = setContents;
+
+                        // qui ci va il link al regolamento
+                        // TODO: in caso questo non venga aggiornato, mettere un link aggiornato
+                        httpRequest.open('GET', 'https://raw.githubusercontent.com/wiki/MozillaItalia/assets/Regolamento.md');
```

Comment:
Così viene stampato un file markdown senza rendering in HTML però fa la richiesta e basta credo che il codice sia incompleto.

Reply:
l'hai provato?